core/arch/arm/plat-imx/registers/imx-regs.h

```diff
@@ -0,0 +1,42 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2015 Freescale Semiconductor, Inc.
+ * All rights reserved.
+ * Copyright (c) 2016, Wind River Systems.
+ * All rights reserved.
+ * Copyright 2017-2018 NXP
+ *
+ */
+#ifndef PLAT_IMX_IMX_REGS_H
+#define PLAT_IMX_IMX_REGS_H
+
+#ifdef CFG_MX6
+#include <registers/imx6-regs.h>
+#elif defined(CFG_MX7)
+#include <registers/imx7-regs.h>
+#elif defined(CFG_MX7ULP)
+#include <registers/imx7ulp-regs.h>
```

Comment:
This patch is not only a cleanup, but adds new things?

Reply:
we have only introduce the registers definition . nothing is compiled.
we will remove them and reintroduce with the soc support then.